hologram_dashboard/fetcher/hologram_availability_fetcher.cc

```diff
@@ -0,0 +1,34 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+#include "hologram_availability_fetcher.h"
+using namespace wireless_android_play_analytics;
```

Comment:
I think you should wrap code below with

namespace wireless_android_play_analytics {


}  // namespace wireless_android_play_analytics

Reply:
Do I need it for both header file and implementation?

Reply:
Done